engine/gui/src/gui.cpp

```diff
@@ -4269,6 +4270,19 @@ namespace dmGui
         out_n->m_Node.m_ResetPointProperties = 0;
         if (n->m_Node.m_Text != 0x0)
             out_n->m_Node.m_Text = strdup(n->m_Node.m_Text);
+        
+        // Handle render constants - clone them if callback is available and source has them
+        if (n->m_Node.m_RenderConstants && scene->m_CloneRenderConstantsCallback)
+        {
+            out_n->m_Node.m_RenderConstants = scene->m_CloneRenderConstantsCallback(n->m_Node.m_RenderConstants);
+            out_n->m_Node.m_RenderConstantsHash = n->m_Node.m_RenderConstantsHash;
+        }
+        else
+        {
+            // Ensure cloned nodes don't share the pointer if no callback available
```

Comment:
I am looking at this diff on a phone which is not optimal, but under what circumstances is no callback available?

Reply:
I removed this comments